### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by prabhu**
A new project at https://dev.azure.com/shiftleftsecurity/sl-appthreat/ will be used for demonstrating ShiftLeft, AppThreat scanning for this repo.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- task: Maven@3
+  inputs:
+    mavenPomFile: 'pom.xml'
+    mavenOptions: '-Xmx3072m'
+    javaHomeOption: 'JDKVersion'
+    jdkVersionOption: '1.8'
+    jdkArchitectureOption: 'x64'
+    publishJUnitResults: false
+    goals: 'compile'
+- script: docker pull quay.io/appthreat/sast-scan
```
Comment (tuxology) [inline azure-pipelines.yml:22]: SHould we push these images to Shiftleft Dockerhub?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  displayName: "ShiftLeft Scan"
```
Comment (tuxology) [inline azure-pipelines.yml:23]: Maybe call it `ShiftLeft AppThreat Scan`?

**Review by tuxology — APPROVED** [review]
Just some nitpicks

File: azure-pipelines.yml (hunk 1 above)
Comment (prabhu) [reply azure-pipelines.yml:22]: Yes working on Jenkinsfile for that. We can use the existing one meanwhile.

File: azure-pipelines.yml (hunk 2 above)
Comment (prabhu) [reply azure-pipelines.yml:23]: Suggested the product name as `ShiftLeft Scan`.

**Pull request merged**